hazelcast-sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -486,12 +486,21 @@ SqlAlterJob SqlAlterJob() :
     SqlParserPos pos = getPos();
 
     SqlIdentifier name;
-    SqlAlterJob.AlterJobOperation operation;
+    SqlNodeList sqlOptions = SqlNodeList.EMPTY;
+    SqlAlterJob.AlterJobOperation operation = null;
 }
 {
     <ALTER> <JOB>
     name = SimpleIdentifier()
     (
+        <OPTIONS>
```

Comment:
in design doc it was proposed to be `ALTER JOB MODIFY OPTIONS`. Was it changed to `ALTER JOB OPTIONS`?

Reply:
`MODIFY` seems redundant since the statement already contains `ALTER`.

Reply:
some SQL variants have for example `ALTER TABLE table_name ALTER [COLUMN] column_name ...`. @viliam-durina WDYT?

Reply:
I'm fine with `ALTER JOB <name> OPTIONS ...`. I also like that it can optionally follow with the RESUME option.